lms/static/templates

```diff
@@ -0,0 +1 @@
+../templates
```

Comment:
why is it required here ?

Reply:
That is the same structure followed in `lms/static/templates --> lms/templates` base folder